core/src/main/java/hudson/model/Run.java

```diff
@@ -2422,7 +2422,7 @@ public String getEntryID(Run e) {
 
     private static class DefaultFeedAdapter implements FeedAdapter<Run> {
         public String getEntryTitle(Run entry) {
-            return entry.getDisplayName()+" ("+entry.getBuildStatusSummary().message+")";
+            return entry.getFullDisplayName();
```

Comment:
So my first commit  https://github.com/jenkinsci/jenkins/pull/2877/commits/aec91c4f7bff8ed89eb496321bdc2d61a35fc135 would be the right choice then?
Or is there a more appropriate/elegant solution?